hw/mcu/nordic/nrf52xxx/src/nrf52_periph.c

```diff
@@ -33,7 +33,7 @@
 #include "bus/drivers/i2c_hal.h"
 #endif
 #endif
-#if MYNEWT_VAL(SPI_0_MASTER) || MYNEWT_VAL(SPI_1_MASTER) || MYNEWT_VAL(SPI_2_MASTER)
+#if MYNEWT_VAL(SPI_0_MASTER) || MYNEWT_VAL(SPI_1_MASTER) || MYNEWT_VAL(SPI_2_MASTER) || MYNEWT_VAL(SPI_2_MASTER)
```

Comment:
Shouldn't SPI_2_MASTER be SPI_3_MASTER above ?

Reply:
Correct, fix pushed. Thanks! Any insight into the failing travis checks?